docs/source/partition-function.rst

```diff
@@ -0,0 +1,226 @@
+   N_a\langle \sigma v \rangle_m &=& N_a\langle \sigma v \rangle_m' \dfrac{G_i}{G_m}
+   \end{eqnarray}
+   
+where the quantites :math:`\lambda_{\gamma}'` and :math:`N_a\langle \sigma v \rangle_m'` are provided by REACLIB, under the ``-v`` flag.
+   
+Implementing Partition Functions
+--------------------------------
+
+The partition function information is contained in three main classes:
+
+* :func:`PartitionFunction <pynucastro.nucdata.PartitionFunction>` materialize the temperature and the partition function values into an object, interpolating across all the defined points using a cubic spline interpolation. If a temperature value is outside the temperature range, we keep its value constant to the nearest boundary value. 
+
+* :func:`PartitionFunctionTable <pynucastro.nucdata.PartitionFunctionTable>`  reads a table and construct a dictionary between each nucleus and their partition function class object.
+
+* :func:`PartitionFunctionCollection <pynucastro.nucdata.PartitionFunctionCollection>` collects all the formatted table information, inside ``/nucdata/PartitionFunction/``. It allow us to include the high temperature tables in [2] and to select the model used to compute the partition functions, respectively. By default, we include high temperatures, and our partition function model to be the *finite range droplet model (FRDM)*. If a nucleus is not in the collection, we set the partition function values to 1.0 by default.
+
+Inside the ``Nucleus`` class, we have defined :func:`.set_partition_function()` which setup our partition function collection, our high temperatures consideration, and the model used to compute the partition function data. On the other hand, :func:`.get_partition_function()` assigns a partition function class object to the defined nucleus. Let us illustrate how it work:
+
+.. code-block:: python
+   
+   import pynucastro
+   
+   co46 = pynucastro.rates.Nucleus('co46')
+   pCollection = pynucastro.nucdata.PartitionFunctionCollection()
+
+   co46.set_partition_function(pCollection=pCollection, set_data='etfsiq', use_high_temperatures=True)
+   pf_co46 = co46.get_partition_function()
+
+Now, from this point we define a method inside a :func:`Rate <pynucastro.rates.Rate>` named :func:`.set_partition_function()` which reads a partition function collection
+and setup all the nucleus inside the reaction rate. Let us illustrate now, how it works:
+
+.. code-block:: python
+
+   import pynucastro
+
+   pCollection = pynucastro.nucdata.PartitionFunctionCollection()
+
+   o18_pg_f19 = pynucastro.rates.Rate('../library/o18-pg-f19-il10')
+   o18_pg_f19.set_partition_function(pCollection=pCollection, set_data='frdm', use_high_temperatures=True)
+
+   p = o18_pg_f19.reactants[0]
+   o18 = o18_pg_f19.reactants[1]
+   f19 = o18_pg_f19.products[0]
+
+   pf_p = p.get_partition_function()
+   pf_o18 = o18.get_partition_function()
+   pf_f19 = f19.get_partition_function()
+
+The next step is to apply 
+
```

Comment:
seems like there should be something more here?

Reply:
ohh yes, I forgot to erase that portion